scene/animation/animation_player.cpp

```diff
@@ -455,11 +457,11 @@ void AnimationPlayer::play_section(const StringName &p_name, double p_start_time
 
 	Playback &c = playback;
 
-	if (c.current.from) {
+	if (!c.current.animation_name.is_empty()) {
```

Comment:
This change caused issue https://github.com/godotengine/godot/issues/113165, so it was reverted in https://github.com/godotengine/godot/pull/113187.

As I commented that, but explain it here again: Avoidance hold reference is fine, but change of the animation existence check to use animation name was wrong since current animation uses empty and [stop] as the stopped state. See also https://github.com/godotengine/godot/pull/98704.